### DISCUSSION TITLE
Option to add specific quote lengths by word count

### DISCUSSION THREAD

**Discussion by rvjanapin** [monkeytypegame/monkeytype#1323] [Ideas]
I find the quote categories too broad , some quotes are too short and too long.
Maybe adding a custom option where you can specify the range of quote lengths ex. quotes that have word counts of 20-30.

I see the problem that more quotes will repeat themselves when given a very specific word range so maybe have a minimum range  of maybe 5 or 10 words so that it still will be more specific lengths but lessen quotes repeating

**Comment by musjj** [comment]
Yeah, would love to see this too. `medium` feels too short for me and `long` feels too long. A more specific filter would be nice.

**Comment by Leonabcd123** [comment]
You can filter quotes by a custom length in the quote search modal: `quote` -> search icon -> `filter by length` -> `custom` -> enter minimum and maximum number of words.
